app/MomMock/Entity/Package.php

```diff
@@ -0,0 +1,203 @@
+<?php declare(strict_types=1);
+
+namespace MomMock\Entity;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\ConnectionException;
+use Doctrine\DBAL\Exception\InvalidArgumentException;
+use Doctrine\DBAL\Query\QueryBuilder;
+
+use function array_map as map;
+
+class Package extends AbstractEntity
+{
+    const TABLE_NAME = 'shipping_package';
```

Comment:
DocBlock missing.

Reply:
done

Reply:
To explain a little, it's not possible to supply type information for static code analysis for constants (e.g. `@var string` or `@const string` has no effect).
Also, a description like `Table name` seems useless.